nodejs/week2/src/backend/routes/reservations.js

```diff
@@ -0,0 +1,21 @@
+//Bringing dependencies
+const express = require('express');
+const app = express();
+const reservations = require("../data/reservations");
+
+//JSON response for all reservations
+app.get('/reservations', (req,res)=>{
+    res.send(reservations);
+});
+
+//JSON response for reservations with id
+app.get('/reservations/:id', (req,res)=>{
+    const id = parseInt(req.params.id);
```

Comment:
what happens if id is not a valid number?
what happens if there's no reservation with that id?